google-cloud-container/src/test/java/com/google/cloud/container/v1/it/Util.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.container.v1.it;
+
+import com.google.cloud.container.v1.ClusterManagerClient;
+import com.google.container.v1.Cluster;
+import com.google.container.v1.ListClustersResponse;
+import java.io.IOException;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public class Util {
+  // Cleans existing test resources if any.
+  private static final int DELETION_THRESHOLD_TIME_HOURS = 24;
+
+  /** tear down any clusters that are older than 24 hours * */
+  public static void cleanUpExistingInstanceCluster(
+      String projectId, String zone, ClusterManagerClient client)
+      throws IOException, ExecutionException, InterruptedException {
+
+    ListClustersResponse clustersResponse = client.listClusters(projectId, zone);
+    List<Cluster> clusters = clustersResponse.getClustersList();
+
+    for (Cluster cluster : clusters) {
+      if (isCreatedBeforeThresholdTime(cluster.getCreateTime())) {
+        client.deleteCluster(projectId, zone, cluster.getName());
+      }
+    }
+  }
+
+  public static boolean isCreatedBeforeThresholdTime(String timestamp) {
```

Comment:
Can we make this method private?

Reply:
I see it still being `public` in the latest commit. Do we want to stick to it. If it's not  used outside, Id change it to private.